testing/upload_validation.ts

```diff
@@ -1769,6 +1771,44 @@ ${endpointKey ? 'endpointKey is set' : `requiresEndpointUrl is ${requiresEndpoin
             'contextProperties',
           ]);
         }
+
+        if (filterableProperties) {
+          if (filterableProperties.length > 5) {
```

Comment:
Nit, we have an object in this file called `Limits` where we stash constants for all the limits.